### DISCUSSION TITLE
The transaction is executed, but the ether is not replenished

### DISCUSSION THREAD

**Discussion by SavelevaOnClick** [ConsenSys-archive/ganache#3525] [Q&A]
I'm doing: `web3.eth.sendTransaction({from: accounts[0], to: "0xc0c0Ea10d8FABd385C7d9c0CB10a84a6B86258ab", value: "100000000000000"})`;

get: `TypeError: Cannot read property 'stack' of undefined
    at IpcMainImpl.<anonymous> (/webpack:/src/integrations/ethereum/index.js:96:31)`;

the transaction goes through, but the ether has not been transferred; I'm new, can anyone come across?

**Comment by davidmurdoch** [comment]
Hi @SavelevaOnClick. Can you share more code?
